.prettierignore

```diff
@@ -0,0 +1 @@
+CHANGELOG.md
```

Comment:
This is to avoid `prettier` to complain about `*` VS `-` in the changelog generated by `release-please`